labs/fabric-debugger.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: VS Code Plugin for Chaincode Developers
+parent: Labs
+---
+# Lab Name
+VS Code Plugin for Chaincode Developers
+
+
+# Short Description
+A VS code extension and toolbox to debugging a chaincode. The extension once installed can be used by developers of chaincode to run and debug from their IDE, to understand how their chaincode behaves.
+
+# Scope of Lab
+Hyperledger Fabric chaincode developers often have to repeat the same steps to set up their environment and deploy code for debugging. This process can be tedious and time-consuming. If the IDE had a built-in debugging interface, it could automate these tasks, making the developer's job easier and faster.
+
+# Initial Committers
+- [Arun S M](https://github.com/arsulegai)
+- [Kent Lau](https://github.com/kentydotcom)
+- [Claudia](https://github.com/technomad01)
+- [Chinmayi D S](https://github.com/urgetolearn)
+- [Varsha](https://github.com/varshapichandi30)
+
+# Sponsor
+- [Arun S M](https://github.com/arsulegai) - Vice Chair, TOC
+
+# Pre-existing repository

```

Comment:
Thank you for submitting the proposal. I don't see any code in the existing repo, except README file ?

Reply:
We can create a new repository if there is no code available as of yet. This is a somewhat easy process.

Reply:
Thank you @nidhi-singh02 @tkuhrt , I had many developers interested in making contributions and using the tool when I first proposed the idea (well before the mentorship program). This repository was created to capture their interest and keep them updated on any further developments. I propose transferring the repository to retain their interest. Currently, the mentees have spent a few weeks on research tasks, and now they are in the build phase. I am open to your further suggestions.

Reply:
Sounds great! I was under the assumption that some code needs to be there for Lab proposal. Thanks for clarifying Tracy.